### ISSUE TITLE
Installing AutoMapper in a Profile Based PCL (Profile 111 package.config)

### ISSUE THREAD

**Issue by IlSocio** [AutoMapper/AutoMapper#1532]
When I try to install AutoMapper 5.0.2 in a PCL Profile 111 which is using package.config I get the error below:
`Could not install package 'System.Runtime.InteropServices.RuntimeInformation 4.0.0'. You are trying to install this package into a project that targets '.NETPortable,Version=v4.5,Profile=Profile111', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.`

In order to install AutoMapper I'm forced to use the new project.json in my PCL 
But I can't because it is not yet supported by Xamarin Studio (and I need Xamarin Studio to compile for iOS)

**Comment by jbogard** [comment]
Are you on the Alpha channel for Xamarin?

And yeah, PCL is on its way out too in favor of NET Standard, but not all the tooling is there yet.

**Comment by IlSocio** [comment]
No, I'm using the stable channel.

**Comment by IlSocio** [comment]
I just created the repository below to reproduce the issue:
https://github.com/IlSocio/HumanizerTest

Try to install Automapper in PCL_111 project and you'll see the error.
Despite it has been successfully installed in PCL_111_json and in PCL_NetStandard_1.1

**Comment by jbogard** [comment]
Yeah, I see it. Spiking out a package with Profile111

**Comment by jbogard** [comment]
I'm going to push out a MyGet with Profile111, I'll ping back when it's up

**Comment by jbogard** [comment]
It's up on the MyGet, can you try it out?

**Comment by IlSocio** [comment]
Hi Jimmy, I can't test the functionalities, but I can confirm that the package gets installed properly.
Thanks :+1:

**Comment by jbogard** [comment]
Awesome!

**Comment by seanfisher** [comment]
@jbogard I'm having the same issue with the same `System.Runtime.InteropServices.RuntimeInformation 4.0.0`. I'm pretty novice in all of this PCL vs NET Standard confusion. I'm trying to install from NuGet a package that uses Automapper 5 as its own dependency. My project is Xamarin PCL profile 111. I have been using Xamarin Studio beta 6.0.1 but I'm currently upgrading to alpha so see if it helps. Do you have any steps for getting around this?

**Comment by jbogard** [comment]
Use the MyGet build.

**Comment by seanfisher** [comment]
So if I install Automapper 5 myself through the MyGet will that help when I install my dependency which in turn depends on Automapper 5 through the regular NuGet?

**Comment by jbogard** [comment]
No, you'd have to upgrade the package from there.

**Comment by seanfisher** [comment]
OK thanks. Sorry, I'm just trying to grasp all the moving parts here. Are you then going to push the MyGet PCL profile 111 to NuGet sometime in the future, or will it permanently be on MyGet just for those that need the PCL? I'm looking into updating our Xamarin projects to support NET standard, but I don't see a lot of documentation on how to do that so it might be a while, but I assume once I do all of these problems go away and I should be able to install the NuGet version of Automapper?

**Comment by jbogard** [comment]
Yeah, it's slated for the 5.1 release of AutoMapper. I plan on doing the
dot releases every month or two.

**Comment by brucewilkins** [comment]
Pulling the prerelease package from myget kills our CI. What is the ETA on production build of 5.1?

**Comment by seanfisher** [comment]
I haven't tried it yet, but Xamarin [just announced](https://blog.xamarin.com/net-standard-library-support-for-xamarin/) that they support consuming .NET Standard libraries as of version 6.0.2, which should theoretically fix this issue (for those who use Xamarin).

**Comment by jbogard** [comment]
@brucewilkins 5.1.1 is out!

And I will shortly be removing Profile 111 support, now that Xamarin supports .NET Standard. It'll be gone in 5.2.
